src/js/core/util/animation/animation.js

```diff
@@ -283,11 +283,49 @@
 				}
 			};
 
+			/**
+			 * Method resets startTime for each animations to current time
+			 * @private
+			 * @param {*} animateConfig
+			 */
+			function resetStartTimeForAnimateConfig(animateConfig) {
```

Comment:
I am getting the exception during scrolling on device:
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at resetStartTimeForAnimateConfig (tau.js:18923)
    at Animate.prototype.reset (tau.js:18943)
    at Marquee.prototype.reset (tau.js:19711)
    at Listview.prototype._onChange (tau.js:32417)
    at Listview.prototype.handleEvent (tau.js:32683)
    at Object.trigger (tau.js:1273)
    at Listview.prototype.trigger (tau.js:8119)
    at Listview.prototype._rollDown (tau.js:32075)
    at Listview.prototype._onRotary (tau.js:32133)